arrocke/gloss-translation
# feat: verify emails and handle bounces and complaints

## What has changed



* Added an invitation key that can be used to complete sign up
* Email is considered verified upon completing the profile form.
* Added route to handle SNS messages from SES when an email bounces or has a complaint.
* Users who have bounced, complained, or not accepted the invite cannot receive email.

## Connected Issues



closes #110, #115

## QA Steps



1. Log in to AWS
2. Navigate to https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/verified-identities/globalbibletools.com/send-test-email
3. Send bounced or complaint email
4. Check email status in users view as an admin

## Post-Deployment



- Set up SNS subscriber in production
